cmd/wrapcontract/wrapcontract.go

```diff
@@ -1,33 +1,96 @@
 package wrapcontract
 
 import (
-	"fmt"
 	_ "embed"
-	"github.com/spf13/cobra"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"encoding/json"
+
 	"github.com/0xPolygon/polygon-cli/util"
+	"github.com/spf13/cobra"
 )
 
 var (
 	//go:embed usage.md
-	usage                  string
+	usage string
+	json_storage *string
```

Comment:
nit: go naming convention

```suggestion
	jsonStorage *string
```

Same for `deployed_bytecode` and `storage_bytecode` :)

Reply:
a shame it does not match with python convention to which I'm used :') changed now!